src/cmdlib.sh

```diff
@@ -255,6 +255,33 @@ EOF
     return "$(cat "${workdir}"/tmp/rc)"
 }
 
+openshift_git_hack() {
+    # When OPENSHIFT_GIT_HACK is defined as a build environment variable,
+    # this will fetch the missing .git into the directory where its expected.
+
+    # Openshift 3.9 does not expose the GIT Repo.
+    # See https://github.com/coreos/coreos-assembler/issues/320
+    # TODO: Remove this when https://github.com/openshift/source-to-image/issues/457
```

Comment:
that issue is closed so maybe we need more guidance on when we can remove this (i.e. when based on Openshift XX with S2I YY)

Reply:
The change logs are not really helpful on either Openshift or S2I. Still hunting...

Reply:
It looks like version v1.0.6 of the Source-to-Image has the change However, I can't find which version of Openshift has that in the default `ose-docker-builder` container.

Reply:
guess we could pull the container and find out

Reply:
One would think so:
```
ben@porpentina:~ $ sudo podman run --rm -it openshift3/ose-docker-builder:v3.10.45-5 version
openshift-docker-build v3.10.45
```
That's not the case.